packages/api/src/external/commonwell/cq-bridge/coverage-enhancement-init.ts

```diff
@@ -35,18 +38,41 @@ export async function initEnhancedCoverage(
     }
     return getPatientsToEnhanceCoverage(cxIds);
   };
+  let patients = await getPatientsToProcess();
 
-  const patients = await getPatientsToProcess();
-  const patientsByCx = groupBy(patients, "cxId");
+  // TODO run the same relinking logic on any failed patients?
+  // for patient in patients, if cw link status is failed, notify sentry
+  let failedPatients: (PatientToLink | Patient)[] = [];
+  if (patients.length > 0 && patients[0] !== undefined) {
```

Comment:
filtering out patients that have linking failed for CE. 
Not waiting on patients in `processing` state since they will get captured in the next 20m train. 
Not relinking patients in `failed` state since they will hold up whole EC train. Lets just stick with notifying sentry and see how often there is a problem here

Reply:
Looking at first patient to determine whether its of type PatientToLink or Patient. Will make those other changes!

Reply:
I simplified things a lot and implemented the suggested changes, partially by removing the PatientToLink type.